content/docs/details/backend-reading-allow-list/index.md

```diff
@@ -0,0 +1,26 @@
+---
+title: Accessing Public resources from the Backend
+publishedDate: '2021-04-26T21:00:00.0Z'
+lastUpdated: '2022-04-26T08:33:00.0Z'
+description: The Backstage backend restricts what resources can be fetched by default. This describes how to add sources which can be accessed.
+---
+
+## Introduction
+
+The Backstage backend restricts what resources can be fetched by default. Only urls matching `backend.reading.allow` in 
+Backstage config can be read. This guide describes how to configure this list. This would allow you to do something like 
+fetch a public API definition in a catalog info file.
+
```

Comment:
Are there default items in this config so that the reader can access things like GitHub? If so, could we list the defaults here since they don't appear in the UI?

Reply:
Somewhat confusingly github and other integrations with their own readers don't use this list. This is only for public resources. There is *.example.com and *.mozilla.org out of the box I can document those (or we might be better off removing them?)